pkg/prowgen/testdata/zz_fixture_TestGenerateJobBase_config_variant.yaml

```diff
@@ -5,7 +5,3 @@ decoration_config:
 labels:
   ci-operator.openshift.io/variant: whatever
 name: pull-ci-org-repo-branch-whatever-test
-spec:
-  containers:
-  - name: test
-    resources: {}
```

Comment:
Specs are irrelevant when testing `JobBases` so they don't need to be in the fixtures